api/queries_pr.go

```diff
@@ -637,15 +637,23 @@ func AddPullRequestReviews(client *Client, repo ghrepo.Interface, prNumber int,
 		return nil
 	}
 
+	// The API requires empty arrays instead of null values
+	if len(users) == 0 {
+		users = []string{}
+	}
+	if len(teams) == 0 {
+		teams = []string{}
+	}
```

Comment:
Since we're doing this to avoid `nil` slices, I think it'd be clearer to check against `nil` instead of length. What do you think?

```suggestion
	if users == nil {
		users = []string{}
	}
	if teams == nil {
		teams = []string{}
	}
```